format-r-code/action.yaml

```diff
@@ -0,0 +1,73 @@
+name: "Format R code with air"
+description: "Automatically format R code with air."
+author: "Damon Bayer, Garrick Aden-Buie"
+
+inputs:
+  version:
+    description: Version of air, e.g. `0.1.2`.
+    default: latest
```

Comment:
Perhaps, pick the latest version and leave it as default? I'm thinking ahead if they make breaking changes.